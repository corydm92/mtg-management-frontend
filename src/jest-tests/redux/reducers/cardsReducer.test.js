import cardsReducer from '../../../redux/reducers/cardsReducer';

describe('Cards Reducer Test', () => {
  it('should return the initial state', () => {
    expect(cardsReducer(undefined, {})).toMatchSnapshot();
  });
});
