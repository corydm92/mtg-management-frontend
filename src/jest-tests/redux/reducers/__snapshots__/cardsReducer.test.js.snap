// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards Reducer Test should return the initial state 1`] = `
Object {
  "entities": Object {},
  "result": Array [],
}
`;
